tests/test_with_credential.rs

```diff
@@ -1,35 +1,52 @@
+mod util;
+
 use rspotify::{
     model::{AlbumId, AlbumType, ArtistId, Country, Market, PlaylistId, TrackId, UserId},
     prelude::*,
-    ClientCredsSpotify, Credentials,
+    ClientCredsSpotify,
 };
 
 use maybe_async::maybe_async;
 
+#[cfg(target_arch="wasm32")]
```

Comment:
Added to remove a warning on unused import when building for other architectures